dev/ProgressRing/ProgressRingAutomationPeer.h

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "ProgressRing.h"
+#include "ProgressRingAutomationPeer.g.h"
+
+class ProgressRingAutomationPeer :
+    public ReferenceTracker<ProgressRingAutomationPeer, winrt::implementation::ProgressRingAutomationPeerT, winrt::IRangeValueProvider>
```

Comment:
Do we not get IRangeValueProvider from RangeBaseAutomationPeer ?

Reply:
removed